tests/extensions/copydb.sh

```diff
@@ -33,6 +33,20 @@ create extension intarray cascade;
 create extension postgis cascade;
 EOF
 
+# Create a few tables with foreign keys to ensure that the query
+# with order FK relationships works without issues.
+psql -a ${PGCOPYDB_SOURCE_PGURI} <<EOF
+create table one (id int, two_id int);
+create table two (id int, one_id int, three_id int);
+create table three (id int);
+alter table one add constraint one_pk primary key (id);
+alter table two add constraint two_pk primary key (id);
+alter table three add constraint three_pk primary key (id);
+alter table one add constraint one_two_fk foreign key (two_id) references two(id);
+alter table two add constraint two_one_fk foreign key (one_id) references one(id);
+alter table two add constraint two_three_fk foreign key (three_id) references three(id);
```

Comment:
Can we inline the pkey and fkey definitions into the CREATE TABLE statements? I think it's easier to read. It needs be, let's just add another SQL file to that test unit (such as `ddl.sql` as found in other units) to have more flexibility in the SQL syntax/layout.